scripts/make_metacal_catalog.py

```diff
@@ -0,0 +1,188 @@
+"""
+make_metacal_catalog.py
+
+Save catalogs to parquet from metacal runs across available patches.
+"""
+import os
+import re
+
+import numpy as np
+from astropy.table import hstack
+
+from lsst.daf.persistence import Butler
+from lsst.daf.persistence.butlerExceptions import NoResults
+
+
+def _ensure_butler_instance(butler_or_repo):
+    if not isinstance(butler_or_repo, Butler):
+        return Butler(butler_or_repo)
+    return butler_or_repo
+
+
+def generate_metacal_catalog(output_dir, butler, tract, patches=None,
+                             overwrite=True, verbose=False,
+                             filename_prefix='metacal',
+                             parquet_engine='pyarrow',
+                             **kwargs):
+    """Save catalogs to parquet from individual metacal patches.
+    Iterates through patches, saving each in append mode to the save parquet file.
```

Comment:
Iterates through patches, saving each in append mode to the same parquet file.

Reply:
You are right sorry. At some point I thought it would append all the patches provided as args. It would be more logical IMHO, but it can be discussed later